### ISSUE TITLE
Released gem is throwing Syntax Error

### ISSUE THREAD

**Issue by kiafaldorius** [hybridgroup/taskmapper-unfuddle#1]
```
SyntaxError: /opt/local/lib/ruby/gems/1.8/gems/unfuddler-0.0.3/lib/unfuddler.rb:34: syntax error, unexpected '=', expecting '|'
      define_method(method) do |url, data = nil|
                                           ^
/opt/local/lib/ruby/gems/1.8/gems/unfuddler-0.0.3/lib/unfuddler.rb:115: syntax error, unexpected '=', expecting '|'
.....-4]) + "!") do |resolution = {}|
                              ^
/opt/local/lib/ruby/gems/1.8/gems/unfuddler-0.0.3/lib/unfuddler.rb:145: syntax error, unexpected kEND, expecting $end
    from /opt/local/lib/ruby/site_ruby/1.8/rubygems/custom_require.rb:31:in `gem_original_require'
    from /opt/local/lib/ruby/site_ruby/1.8/rubygems/custom_require.rb:31:in `require'
    from /opt/local/lib/ruby/gems/1.8/gems/ticketmaster-unfuddle-0.0.0/lib/ticketmaster-unfuddle.rb:1
    from /opt/local/lib/ruby/site_ruby/1.8/rubygems/custom_require.rb:36:in `gem_original_require'
    from /opt/local/lib/ruby/site_ruby/1.8/rubygems/custom_require.rb:36:in `require'
    from (irb):4
```

This is on ruby 1.8.7 (2010-01-10 patchlevel 249) [i686-darwin10]

**Comment by sirupsen** [comment]
Thanks for the report. I'll look into it.

Would seem like 1.8.7 doesn't support default params like that via define_method.

**Comment by sirupsen** [comment]
Fixed in 0.0.4

http://github.com/Sirupsen/unfuddler/commit/e65a9ab01a21dbfb57b2f5b83326e1a10203eb24
